src/components/SuggestedValidators/SuggestedValidators.js

```diff
@@ -36,7 +36,7 @@ const SuggestedValidators = (props: SuggestedValidatorsProps) => {
 			</Helmet>
 			<Route exact path='/suggested-validators'>
 				<Box m={4} mt={10}>
-					<Link m={4}>
+					<Link href='/#/returns-calculator' m={4}>
```

Comment:
Use hash router's way instead of this.

Useful resource: https://reacttraining.com/react-router/web/api/HashRouter

Reply:
Here, if we are using hash router's link, the underline icon is rendering, that's why i prefer to use chakra link, so that UI will remain same. Let me know your views on it.

Reply:
@suyanksaraswat 

> the underline icon is rendering

I'm not sure what this means - could you add a SS?

Reply:
![Screenshot (16)](https://user-images.githubusercontent.com/31132741/81291445-62281480-9087-11ea-9463-f34daf9f87e6.png)

Reply:
@suyanksaraswat Got it, the underline isn't required - use hash router